source/adapters/cuda/tracing.cpp

```diff
@@ -22,14 +22,11 @@
 #include <iostream>
 
 #ifdef XPTI_ENABLE_INSTRUMENTATION
-constexpr auto CUDA_CALL_STREAM_NAME = "sycl.experimental.cuda.call";
-constexpr auto CUDA_DEBUG_STREAM_NAME = "sycl.experimental.cuda.debug";
+constexpr auto CUDA_CALL_STREAM_NAME = "ur.adapter.cuda.call";
```

Comment:
in the case of UR, the stream name is just [`ur`](https://github.com/oneapi-src/unified-runtime/blob/main/source/loader/layers/tracing/ur_tracing_layer.cpp#L23). For consistency, should we call this one `ur.adapter.cuda` or rename the ur one to `ur.call` ? I have no preference.